clang/test/SemaSYCL/Inputs/sycl.hpp

```diff
@@ -101,9 +101,8 @@ struct DeviceValueType<dataT, access::target::local> {
 
 template <typename dataT, int dimensions, access::mode accessmode,
           access::target accessTarget = access::target::global_buffer,
-          access::placeholder isPlaceholder = access::placeholder::false_t,
-          typename propertyListT = ONEAPI::accessor_property_list<>>
-class accessor {
+          access::placeholder isPlaceholder = access::placeholder::false_t>
```

Comment:
I think you accidentally removed `typename propertyListT = ONEAPI::accessor_property_list<>`

Reply:
@elizabethandrews  That's the next thing I am trying to understand. Do all 3 classes (accessor, sampler and stream) have a property_list? What's the purpose of it?